packages/perseus/src/widgets/radio/choice-indicator.module.css

```
@@ -0,0 +1,104 @@
+.base {
+    appearance: none;
+    background-color: var(--wb-semanticColor-core-transparent);
+    border: solid 0.2rem var(--wb-semanticColor-core-border-neutral-strong);
```

it's unfortunate that the border values in WB is still not very detailed e.g. like we can use the `--wb-border-width-medium` but the other option we have based on the docs is using the sizing as mentioned in the [docs](https://khan.github.io/wonder-blocks/?path=/docs/packages-tokens-sizing--docs) it can be used for border width.
```suggestion
    border: solid var(--wb-sizing-size_020) var(--wb-semanticColor-core-border-neutral-strong);
```